arbnode/dataposter/data_poster.go

```diff
@@ -802,6 +819,9 @@ func DataPosterConfigAddOptions(prefix string, f *pflag.FlagSet, defaultDataPost
 	f.Bool(prefix+".use-db-storage", defaultDataPosterConfig.UseDBStorage, "uses database storage when enabled")
 	f.Bool(prefix+".use-noop-storage", defaultDataPosterConfig.UseNoOpStorage, "uses noop storage, it doesn't store anything")
 	f.Bool(prefix+".legacy-storage-encoding", defaultDataPosterConfig.LegacyStorageEncoding, "encodes items in a legacy way (as it was before dropping generics)")
+	f.String(prefix+".max-fee-cap-formula", defaultDataPosterConfig.MaxFeeCapFormula, "mathematical formula to calculate maximum fee cap")
```

Comment:
Please add comment with the documentation to the syntax  of this expression (https://github.com/Knetic/govaluate)

Reply:
there is no syntax as such except for the fact that the mathematical formula should be supplied as a string and use basic math symbols. But I can add https://github.com/Knetic/govaluate/blob/master/MANUAL.md in the description to guide to available operations